templates/taxi/driver_form.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <h1>{{ object|yesno:"Update,Create" }} driver</h1>
+  <form action="" method="post" novalidate>
```

Comment:
The form element is missing an 'action' attribute. You should specify the URL for the form submission to ensure the form data is sent to the correct view. For example, use `{% url 'taxi:driver-create' %}` or `{% url 'taxi:driver-update' pk=object.id %}` depending on the context.